resources/lang/swe.ini

```diff
@@ -3,44 +3,109 @@ language.name = "Swedish"
+warning.differentlevel = "[VARNING] Du editerar en level som du inte är i för tillfället!"
+; commands
+command.info.title = "Information"
+command.limit.current = "Nuvarande gräns: {%0}"
+command.limit.set = "Block change limit satt till {%0}"
+command.setrange.current = "Nuvarande nivå: {%0}"
+command.setrange.set = "Tool range satt till {%0}"
+command.biomeinfo.attarget = "Biome vid mål"
+command.biomeinfo.atposition = "Biome vid position"
+command.biomeinfo.result = "{%0} biomes hittade in markering"
+command.biomeinfo.result.line = "ID: {%0} Namn: {%1}"
+command.biomelist.title = "Biome lista"
+command.biomelist.result.line = "ID: {%0} Namn: {%1}"
+command.brushname.set = "Penselnamn satt till \"{%0}\""
+command.clearclipboard.cleared = "Urklipps rensat"
+command.flip.try = "Försöker spegla urklipp med {%0}"
+command.flip.success = "Urklipp har speglats"
+command.rotate.try = "Försöker rotera urklipp med {%0} grader"
+command.rotate.success = "Urklipp har roterats"
+command.history.cleared = "Historik rensat"
+command.listchunks.found = "{%0} delar hittade i urvalet"
+command.size = "Markeringsstorlek"
+; selection
+selection.pos1.set = "Position 1 satt till X: {%0} Y: {%1} Z: {%2}"
+selection.pos2.set = "Position 2 satt till X: {%0} Y: {%1} Z: {%2}"
+; session
+session.undo.none = "Det finns inget att ångra"
+session.undo.left = "Du kan ångra {%0} fler gånger"
+session.redo.none = "Inget att återskapa"
+session.redo.left = "Du kan återskapa {%0} fler gånger"
+session.brush.added = "Lade till {%0} till sessionen"
+session.brush.deleted = "Raderade {%0} (UUID {%1})"
+session.brush.removed = "Tog bort {%0} (UUID {%1})"
+session.language.set = "Språk satt till {%0}"
+session.language.notfound = "Språk {%0} inte hittat, återställer till standard!"
+; task
+task.copy.success = "Async Kopiering lyckades, tog {%0}, kopierade {%1} block av {%2} totalt."
+task.count.success = "Async Analysering lyckades, tog {%0}"
+task.count.result = "{%0} block hittade av totalt {%1} block"
+task.fill.success = "Async Fyll lyckades, tog {%0}, {%1} block av {%2} ändrade."
+task.replace.success = "Async Ersätta lyckades, tog {%0}, {%1} block av {%2} ändrade."
+task.revert.undo.success = "Async Ånga lyckades, tog {%0}, {%1} block av {%2} ändrade."
+task.revert.redo.success = "Async Återskapa lyckades, tog {%0}, {%1} block av {%2} ändrade."
+; flags
+flags.keepexistingblocks = "Behåll existerande block"
+flags.keepair = "Behåller luft"
+flags.hollow = "Ihåligt"
+flags.hollowclosed = "Ihåligt med stängda ändar"
+flags.natural = "Neutral"
+; tools
+; wand tool
+tool.wand = "Trollspö"
+tool.wand.lore.1 = "Vänsterklicka på ett block för att sätta position 1 av markering"
+tool.wand.lore.2 = "Högerklicka på ett block för att sätta position 2 av markering"
+tool.wand.lore.3 = "Använd //togglewand för att aktivera funktionaliteten"
+tool.wand.disabled = "Trollspöet är avaktiverat. Använd //toggelwand för att aktivera"
```

Comment:
Please don't change `//togglewand`, since it is a command ^^ (sadly per-user command names are yet not possible)